packages/admin-ui/ui/src/domain/product-categories/modals/edit-product-category.tsx

```diff
@@ -98,6 +99,12 @@ function EditProductCategoriesSideModal(
         </div>
         {/* === DIVIDER === */}
```

Comment:
same here, we lost the divider along the way somewhere so the spacing is off 😁 
https://www.figma.com/file/kSfdBTx7djUsXNexVRYIRl/Nested-categories-Q1-2023?node-id=6389-5663&t=M8G5SX7W66k1tn3b-4